zproject/default_settings.py

```diff
@@ -90,9 +90,21 @@
 # Other auth
 SSO_APPEND_DOMAIN: Optional[str] = None
 
+VIDEO_ZOOM_API_URL: str = 'https://api.zoom.us/'
 VIDEO_ZOOM_CLIENT_ID = get_secret('video_zoom_client_id', development_only=True)
 VIDEO_ZOOM_CLIENT_SECRET = get_secret('video_zoom_client_secret')
 
+# Webex Video Call Settings
+VIDEO_WEBEX_API_URL: str = 'https://webexapis.com/'
+VIDEO_WEBEX_CLIENT_ID = get_secret('video_webex_client_id', development_only=True)
+VIDEO_WEBEX_CLIENT_SECRET = get_secret('video_webex_client_secret')
+# Length of alphanumeric password generated when a Webex meeting is created
+VIDEO_WEBEX_PASSWORD_LENGTH = 11
```

Comment:
How did you pick the number 11 here?

Reply:
From this document: https://help.webex.com/en-us/so30qz/Required-Password-Enforcement-for-Webex-Meetings-Events-and-Training-Sessions#:~:text=Webex%20sites%20will%20be%20configured,and%20to%20slow%20channel%20sites specifically 'The default passwords applied will be 11 or more characters, including mixed case, numbers, and special characters.'